app/tasks/job_attributes_fetcher.rb

```diff
@@ -39,10 +39,11 @@ def application_question_set
   end
 
   def apply_with_cheddar
```

Comment:
I think you've both made changes to this file @patanj101 @daniel-sussman - can we be aware of this and discuss reasoning together so it doesn't cause problems later

Reply:
I've changed the method name to #fetch_application_question_set and added a comment for clarity. This method should only be called once (with #build_application_question_set) because it may trigger a new API call to fetch the questions (depending on the ATS). I changed #apply_with_cheddar to use stored data rather than re-calling the API unnecessarily.